<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Version Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileField">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>431</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFileButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveCurrentButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Current</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RetrieveVersionButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Retrieve Version</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetReferenceButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Set As Reference</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowCommentButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Comment</string>
    </property>
   </widget>
   <widget class="QTableView" name="fileView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>421</width>
      <height>411</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QPlainTextEdit" name="warningFrame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="backgroundVisible">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>fileField</zorder>
   <zorder>selectFileButton</zorder>
   <zorder>SaveCurrentButton</zorder>
   <zorder>RetrieveVersionButton</zorder>
   <zorder>SetReferenceButton</zorder>
   <zorder>ShowCommentButton</zorder>
   <zorder>warningFrame</zorder>
   <zorder>fileView</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
